onnxruntime/core/providers/openvino/ov_versions/data_ops.h

```diff
@@ -0,0 +1,69 @@
+#pragma once
+#include <unordered_set>
+
+namespace onnxruntime {
+namespace openvino_ep {
+
+typedef enum version_num { 
+  VERSION_2020_4,
+  VERSION_2021_1,
+  VERSION_2021_2
+} version_id_e;
+
+using ConfirmationFunction = std::function<bool(const Node*, const Provider_InitializedTensorSet&)>;
+
+typedef struct supportedop {
+  version_id_e version;
+  std::vector<std::string> device_type;  
+  std::string optype;
+} supportedop_t;
+
+typedef struct unsupportedopmode {
+  version_id_e ver;
+  std::string dev;
+  ConfirmationFunction func; 
+} unsupportedopmode_t;
+
+class Capability{
+
+private:
+const GraphViewer& graph_viewer;
+version_id_e version_id;
+std::string device_id;
+std::multimap<std::string, unsupportedopmode_t &> _confirmation_map;
+std::vector<supportedop_t> subgraph_supported;
+std::vector<supportedop_t> no_dimension_supported;
+std::set<int> supported_types_vpu; 
+std::set<int> supported_types_cpu;
+std::set<int> supported_types_gpu;
+std::set<int> supported_types_initializer;
+
+protected:
+  virtual void populate_op_mode_supported();
```

Comment:
Naming convention for functions has to be taken care of

Reply:
Resolved